### ISSUE TITLE
Manual Record CRUD

### ISSUE THREAD

**Issue by harrislapiroff** [littleweaver/django-brambling#57]
It'd be good for organizers to have the ability to manually create and edit shop records, I think.

**Comment by melinath** [comment]
From an event:

>Can I process manual payments through Dancerfly? Last year we didn't sell very many cash tickets, but there were a few (around 35/150) due to flash sales and the odd person that struggled to make a card payment. With Eventbrite I can just go in and add those sales, and not only do the cash buyers get an eticket, they are also added to all of the statistical info, and appear on the final printed guest list (generated by Eventbrite). 
>I also did this with last year's guests and complimentary tickets (we give a few away as prizes at other events for promotional reasons) which worked pretty well.

That's what this ticket is for, correct? If so, there are basically the following features we're looking for:

* [ ] Choose items
* [ ] Choose check / cash payment (which will be auto-confirmed) *or* complimentary
* [ ] Fill out attendee info & order email
* [ ] Send a confirmation email to the appropriate person.

**Comment by melinath** [comment]
From #316:

>[M]anual payments should probably mark any current cart as paid and attach the items to the payment. Manual payment page should also display currently-owed amount (and default the payment amount to that.

**Comment by melinath** [comment]
As part of this, now that discounts can't be applied to old orders, that tab doesn't really make sense separately from the manual order process (whatever that ends up looking like.)

**Comment by melinath** [comment]
Is there anything that's part of this ticket that wouldn't be sufficiently covered by opening an incognito window and registering as an anonymous user?

**Comment by melinath** [comment]
I suppose being able to add items manually to an order that already exists. But... I guess I'm not sure this is a thing we actually need right now, compared to some of the other tickets we have.

**Comment by harrislapiroff** [comment]
Yeah, the UD part is not covered. Also being able to create records quickly. Mostly I just think organizers will want to feel in control of their data. We can't anticipate all the ways they're going to want to modify it or all the reasons they're going to need to add or delete records, but it's basically like we're giving them a spreadsheet that they can't edit.

Some examples that I can think of where an organizer might feel a need to manually CRUD some stuff:
 
*taking registrations at the door with cash/check
* deleting test records (maybe test records that exist alongside real records, such that a "clear data" button would be destructive)
* adding a free t-shirt to someone's order post-hoc without having to go through a discount rigmarole
* switching someone to a different pass type (maybe without even changing the cost for that user) without having to coordinate that user repurchasing it
* updating someone's housing data when they send an email to an organizer saying, "oh, actually, I forgot to put down that I..."

**Comment by harrislapiroff** [comment]
Basically, I think that as long as we don't have some sort of comprehensive CRUD, we're basically telling organizers not to think of Dancerfly as their canonical data store. I agree that it might not be a priority for us to work on it now, but I think either this or making it sync nicer with other tools (as in #274) is important.

**Comment by harrislapiroff** [comment]
Another example of a potential need for CRUD https://trello.com/c/G3mtaqmz/460-ticket-substitution
